scripts/perftest/python/configuration.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+import os
+from os.path import join
+from utils import split_rowcol, config_writer
+import sys
+import logging
+
+mat_type = {'dense': 0.9,
+            'sparse': 0.01}
+format = 'csv'
+has_predict = ['GLM', 'Kmeans', 'l2-svm', 'm-svm', 'naive-bayes']
+
+
+def naive_bayes_datagen(matrix_type, mat_shapes, conf_dir):
```

Comment:
I could be missing something obvious here (since I am not very familiar with Python), but it seems like this function `naive_bayes_datagen` has been defined twice (wit the same signature)